src/js/components/LoginForm.js

```diff
@@ -110,7 +110,7 @@ export default class LoginForm extends Component {
 
     let titleNode;
     if (title) {
-      titleNode = <Heading strong={true}>{title}</Heading>;
+      titleNode = <Heading align='center' strong={true}>{title}</Heading>;
```

Comment:
The Box that contains this is already controlled via `align='center'`.  Why is this change needed?